src/dynamics/mpas/dyn_comp.F90

```diff
@@ -203,6 +207,15 @@ subroutine dyn_init(cam_runtime_opts, dyn_in, dyn_out)
         nullify(pio_init_file)
         nullify(pio_topo_file)
 
+        ! Set dynamical core energy formula for use in cam_thermo.
+        energy_formula_dycore = ENERGY_FORMULA_DYCORE_MPAS
+        call mark_as_initialized("total_energy_formula_for_dycore")
```

Comment:
```suggestion
        call mark_as_initialized('total_energy_formula_for_dycore')
```

The MPAS dycore interface and subdriver have been using single quotes for character strings throughout. For consistency, changing double quotes to single quotes would be preferable.